README.md

```diff
@@ -12,17 +12,19 @@
 
 **napari** is a fast, interactive, multi-dimensional image viewer for Python. It's designed for browsing, annotating, and analyzing large multi-dimensional images. It's built on top of `Qt` (for the GUI), `vispy` (for performant GPU-based rendering), and the scientific Python stack (`numpy`, `scipy`).
 
-We're developing **napari** in the open! But the project is in an **alpha** stage, and there will still occasionally be **breaking changes** from patch to patch. You can follow progress on this repository, test out new versions as we release them, and contribute ideas and code.
+We're developing **napari** in the open! But the project is in an **alpha** stage, and there will still likely be **breaking changes** with each release. You can follow progress on this repository, test out new versions as we release them, and contribute ideas and code.
+
+We're working on [in-depth tutorials](https://github.com/napari/napari-tutorials/tree/master/README.md), but you can also quickly get started by looking below.
```

Comment:
I've set up GitHub pages at https://napari.github.io/napari-tutorials/ since all of our files are Markdown if you'd like to change the lint to that

Reply:
fixed, thanks, much nicer!!